lib/features/home_screen/widgets/contact_icon_widget.dart

```diff
@@ -0,0 +1,34 @@
+import 'package:flutter/material.dart';
+import 'package:url_launcher/url_launcher.dart';
+
+import '../../../config/app_ui_config.dart';
+import '../../../theme/app_theme.dart';
+
+class ContactIconWidget extends StatelessWidget {
+  const ContactIconWidget({super.key, required this.url});
+
+  final String url;
+
+  @override
+  Widget build(BuildContext context) {
+    final Uri _url = Uri.parse('https://flutter.dev');
+
+    return GestureDetector(
+      onTap: () async {
+        if (!await launchUrl(_url)) {
+          throw Exception('Could not launch $_url');
```

Comment:
I guess we can handle Exception like that in slightly safer way. Probably better just to log the error or show a toast message.

Reply:
Yep, throwing here seems risky, cause actually not successfull lauch of url is pretty common. There are tens of reasons why url can't be opened and we don't want errors flying around during normal usage scenario. Would opt for some simple logging instead or even ignoring the event.